ODataConnectedService/src/CodeGeneration/V3CodeGenDescriptor.cs

```diff
@@ -16,6 +16,67 @@ namespace Microsoft.OData.ConnectedService.CodeGeneration
 {
     internal class V3CodeGenDescriptor : BaseCodeGenDescriptor
     {
+        const string loadServiceModelAssignment = "this.Format.LoadServiceModel = GeneratedEdmModel.GetInstance;";
+        const string dynamicModelLoader =
```

Comment:
For maintainability, can you please put a comment with a high-level description on how the class works/will be utilized?